packages/@aws-cdk/aws-eks/lib/ami.ts

```diff
@@ -61,25 +61,11 @@ export function nodeTypeForInstanceType(instanceType: ec2.InstanceType) {
  * @see https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html
  */
 const EKS_AMI: {[version: string]: {[type: string]: {[region: string]: string}}} = {
-  '1.10': parseTable(`
```

Comment:
This is going to break existing code that manages *existing* clusters. You can still update those, even though you may not create new ones. So you cannot remove those entries.

Reply:
Fixed